lib/bats-core/test_functions.bash

```diff
@@ -42,6 +72,37 @@ run() {
   IFS=$'\n' lines=($output)
   IFS="$origIFS"
   set "-$origFlags"
+
+  # Show results. Without quotes, multiple lines are glommed together into one
+  if [ -n "$output" ]; then
+    printf "%s\n" "$output"
```

Comment:
Again, this should be part of an eventual trace feature, not of everyday run. I am surprised that this change of behavior does not fail some of our tests already.